### ISSUE TITLE
cei2html.xsl should display `text()` content of `cei:damage`

### ISSUE THREAD

**Issue by GVogeler** [icaruseu/mom-ca#463]
e.g. http://monasterium.net/mom/9656a096-167c-428d-8a7a-d2db8b004452/SMG_3_AA_II_4_g/my-charter
`<cei:damage>anno</cei:damage>` is displayed as `[...]` and should be `[anno]`

**Comment by GVogeler** [comment]
Check if XPath in https://github.com/icaruseu/mom-ca/blob/master/my/XRX/src/mom/app/charter/xsl/cei2html.xsl#L732 selects the correct condition.
Maybe
```
        <xsl:choose>
          <xsl:when test=".//text()/normalize-space()!=''">
            <xsl:apply-templates/>
          </xsl:when>
          <xsl:otherwise><xsl:text>...</xsl:text></xsl:otherwise>
        </xsl:choose>
```
works better?

**Comment by GVogeler** [comment]
XPath in https://github.com/icaruseu/mom-ca/blob/master/my/XRX/src/mom/app/charter/xsl/cei2html.xsl#L732 selects all occurences of `cei:damage`
I would suggest
```
<xsl:choose>
          <xsl:when test=".//text()/normalize-space()!='' or node()">
            <xsl:apply-templates/>
          </xsl:when>
          <xsl:otherwise><xsl:text>...</xsl:text></xsl:otherwise>
        </xsl:choose>
```

**Comment by GVogeler** [comment]
fixed with #472
